templates/cohorts/new_cohort.html

```diff
@@ -195,6 +195,24 @@ <h4 class="panel-title pull-left">Selected Filters</h4>
             </div>
         </div>
 
+        <div class="cohort-info panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">Details</h4>
+            </div>
+            <div class="panel-body">
```

Comment:
How is the width of the inline-detail sections defined? Is it possible, to use the bootstrap columns for this? make them both 6col wide?